# Raw Velodyne16 LIDAR packet.

Header header      
uint8[1206] data 

